array-manipulation-2/src/chunk.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars */
+
+function chunk(array, size) {
+  var newArray = [];
+  var tempArray = [];
+  var rotation = 0;
+  for (var i = 0; i < array.length; i++) {
+    if (size > rotation) {
+      tempArray.push(array[i]);
+      rotation++;
+      continue;
+    } if (rotation === size) {
+      newArray.push(tempArray);
+      rotation = 0;
+      tempArray = [];
+      tempArray.push(array[i]);
+      rotation++;
+      continue;
+    }
```

Comment:
Instead of doing two if statements with a continue statement, you can just do an if elseif statement, the loop will continue regardless.